src/components/retro/ExportDialog.tsx

```
@@ -0,0 +1,177 @@
+  type ExportData,
+  type ExportOptions,
+} from "@/lib/export/retro-export";
+
+const DEFAULT_FILENAME = "retrospective";
+
+// Replace any character not allowed in filenames with '_'
+// This covers Windows and Unix invalid filename characters plus control chars
+function sanitizeFilename(rawName?: string): string {
+  const candidate = rawName ?? DEFAULT_FILENAME;
+  const sanitized = candidate
+    .replace(/[\x00-\x1f\x7f]/g, "") // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)